File: testcases/src/main/java/regexp/Regexp_Res1.java
package regexp;

import javax.ws.rs.GET;
import javax.ws.rs.Path;
import javax.ws.rs.PathParam;
import javax.ws.rs.core.Response;

@Path("/regexp1/a/{month:\\d{4}-\\d{2}}/b")
public class Regexp_Res1 {

    @GET
    public Response test() {
        return Response.status(Response.Status.OK).build();
    }

}

File: testcases_kt/src/main/kotlin/regexp/Regexp_Res1.kt
package regexp

import javax.ws.rs.GET
import javax.ws.rs.Path
import javax.ws.rs.core.Response

@Path("/regexp1_kt/a/{month:\\d{4}-\\d{2}}/b")
class Regexp_Res1 {
    @GET
    fun test(): Response {
        return Response.status(Response.Status.OK).build()
    }
}
